src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -414,7 +414,7 @@ default boolean lootIcons() {
     @ConfigItem(
         keyName = "minLootValue",
         name = "Min Loot value",
-        description = "Minimum value of the loot to notify",
+        description = "Minimum value of the loot for a notification to be sent",
```

Comment:
Clarify that this is the minimum value of an item stack?

Reply:
```suggestion
        description = "The minimum value of an item for a notification to be sent",
```
Would the user recognize `Dragon bolts (unf) x 100` as being a single item, if it was displayed this way?

Reply:
I would imagine this is clear enough yeah